src/TokenUtils.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+import fs from 'fs';
+import https from 'https';
+import axios from 'axios';
+import qs from 'qs';
+import credentials from '../config.json';
+
+function isAccessTokenExpired() {
+	let currentDate = new Date();
+	let expiryDate = new Date(credentials.expiry_date);
+	return expiryDate.getTime() < currentDate.getTime();
```

Comment:
Also maybe do like a 30 second offset. Just incase you check like 1 second before it expires, then it says it is good to go, but when you make the request it fails

Reply:
a 30 second buffer is not effective b/c the API will return the same token until it officially expires. therefore, a 0.5 second may even be too long. something to consider further